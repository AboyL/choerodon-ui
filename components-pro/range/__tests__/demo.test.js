import demoTest from '../../../tests/shared/demoTest';

demoTest('range-pro');
