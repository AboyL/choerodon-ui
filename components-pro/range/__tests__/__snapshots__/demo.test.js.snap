// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components-pro/range/demo/basic.md correctly 1`] = `
<div>
  <label
    class="c7n-pro-range-wrapper"
  >
    <input
      autocomplete="off"
      class="c7n-pro-range"
      max="1"
      min="0"
      step="0.01"
      type="range"
      value="0"
    />
    <div
      class="c7n-pro-range-track"
    >
      <div
        class="c7n-pro-range-draghandle"
        style="left:0"
      />
      <div
        class="c7n-pro-range-selection"
        style="width:0"
      />
    </div>
  </label>
  <span />
  <label
    class="c7n-pro-range-wrapper c7n-pro-range-disabled"
    style="margin:0.2rem 0 0 0"
  >
    <input
      autocomplete="off"
      class="c7n-pro-range"
      disabled=""
      max="100"
      min="0"
      step="5"
      type="range"
      value="20"
    />
    <div
      class="c7n-pro-range-track"
    >
      <div
        class="c7n-pro-range-draghandle"
        style="left:20%"
      />
      <div
        class="c7n-pro-range-selection"
        style="width:20%"
      />
    </div>
  </label>
  <span />
</div>
`;

exports[`renders ./components-pro/range/demo/controlled.md correctly 1`] = `
<div>
  <label
    class="c7n-pro-range-wrapper"
  >
    <input
      autocomplete="off"
      class="c7n-pro-range"
      max="50"
      min="10"
      name="range"
      step="1"
      type="range"
      value="30"
    />
    <div
      class="c7n-pro-range-track"
    >
      <div
        class="c7n-pro-range-draghandle"
        style="left:50%"
      />
      <div
        class="c7n-pro-range-selection"
        style="width:50%"
      />
    </div>
  </label>
  <span />
  <span
    class="c7n-pro-input-wrapper"
    style="margin:0.2rem 0 0 0"
  >
    <label>
      <input
        autocomplete="off"
        class="c7n-pro-input"
        readonly=""
        type="text"
        value="30"
      />
    </label>
  </span>
  <span />
</div>
`;

exports[`renders ./components-pro/range/demo/dataset.md correctly 1`] = `
Array [
  <label
    class="c7n-pro-range-wrapper"
  >
    <input
      autocomplete="off"
      class="c7n-pro-range"
      max="100"
      min="0"
      name="range"
      step="1"
      type="range"
      value="20"
    />
    <div
      class="c7n-pro-range-track"
    >
      <div
        class="c7n-pro-range-draghandle"
        style="left:11.11111111111111%"
      />
      <div
        class="c7n-pro-range-selection"
        style="width:11.11111111111111%"
      />
    </div>
  </label>,
  <span />,
]
`;

exports[`renders ./components-pro/range/demo/verticalRange.md correctly 1`] = `
<div
  style="height:200px"
>
  <label
    class="c7n-pro-range-wrapper c7n-pro-range-vertical"
  >
    <input
      autocomplete="off"
      class="c7n-pro-range"
      max="100"
      min="0"
      step="5"
      type="range"
      value="0"
    />
    <div
      class="c7n-pro-range-track"
    >
      <div
        class="c7n-pro-range-draghandle"
        style="bottom:0"
      />
      <div
        class="c7n-pro-range-selection"
        style="height:0"
      />
    </div>
  </label>
  <span />
</div>
`;
